src/AWS.Deploy.Common/Recipes/Validation/OptionSettingItemValidators/RequiredValidator.cs

```diff
@@ -19,11 +19,11 @@ public class RequiredValidator : IOptionSettingItemValidator
         public Task<ValidationResult> Validate(object input, Recommendation recommendation, OptionSettingItem optionSettingItem)
         {
             var message = ValidationFailedMessage.Replace("{{OptionSetting}}", optionSettingItem.Name);
-            if (input?.TryDeserialize<SortedSet<string>>(out var inputList) ?? false)
+            if (input?.TryDeserialize<SortedSet<string>>(out var inputList) ?? false && inputList != null)
             {
                 return Task.FromResult<ValidationResult>(new()
                 {
-                    IsValid = inputList.Any(),
+                    IsValid = inputList!.Any(),
```

Comment:
Why do we need to suppress here with the null check added above?

Reply:
even though there is a null check on line 22, visual studio is still indicating that this could be null, which is why I suppressed it.